httpbakery/client_test.go

```diff
@@ -219,21 +275,17 @@ func (r *readCounter) Read(buf []byte) (int, error) {
 	return n, err
 }
 
-func newService(c *gc.C, location string, locator bakery.PublicKeyLocator) *bakery.Service {
+func newService(location string, locator bakery.PublicKeyLocator) *bakery.Service {
 	svc, err := bakery.NewService(bakery.NewServiceParams{
 		Location: location,
 		Locator:  locator,
 	})
-	c.Assert(err, gc.IsNil)
+	if err != nil {
```

Comment:
Not a blocker: I would like to know any guidelines on using panic v. passing gc.C in a test builder function like this.